docs/fluvio/cli/benchmark.mdx

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_position: 40
+title: "benchmark"
+description: "benchmark is a tool for running performance tests on Fluvio"
+---
+
+# The `fluvio benchmark` command
+
+The `fluvio benchmark` command is an alias for `fluvio benchmark matrix`. It runs multiple benchmarks sequentially and displays the results of each benchmark configuration.
+
+## Matrix benchmark
```

Comment:
Let's move ##Producer benchmark ahead of Matrix benchmark.

Reply:
Sure, done!